amit/html/Assigment3/Amit3_4.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta hhtp-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>contact</title>
+        <style>
+            body {
+                background-color: thistle;
+                background-size: cover;
+            }
+
+            a:link {
+                color: navy;
+                background-color: transparent;
+                font-style: normal;
+            }
+
+            a:hover {
+                color: palevioletred;
+                background-color: transparent;
+                font-style: italic;
+            }
```

Comment:
add one line space here or remove spaces above make sure its consistent in all files